src/autoscan.h

```diff
@@ -77,8 +77,6 @@ class AutoscanList {
     /// \brief removes the AutoscanDirectory given by its scan ID
     void remove(size_t id);
 
-    int removeByObjectID(int objectID);
-
     /// \brief removes the AutoscanDirectory with the given location
     /// \param location the location to remove
     /// \return the scanID, that was removed; if nothing removed: INVALID_SCAN_ID
```

Comment:
`int remove(fs::path...)` should be gone from the header too.

Reply:
Done.